packages/editor-ui/src/plugins/codemirror/completions/datatype.completions.ts

```diff
@@ -547,6 +591,36 @@ const numberOptions = (input: AutocompleteInput<number>): Completion[] => {
 	const ONLY_INTEGER = ['isEven()', 'isOdd()'];
 
 	if (Number.isInteger(resolved)) {
+		const nowMillis = Date.now();
```

Comment:
Is this a requirement to only allow 30 years in the future? Can we use whole [possible](https://262.ecma-international.org/5.1/#sec-15.9.1.1) range for date here?

Reply:
The problem would become that we would suggest `toDateTime` on any number. The idea here is that we only suggest it when we are pretty sure it's a timestamp (so within a range of now) but the 30y is arbitrary.